consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -79,7 +79,7 @@ pub struct ByzantineLedgerWorker<
     // processing times for relayed values since we want to track the time from when the network
     // first saw a value, and not when a specific node saw it.
     pending_values: Vec<TxHash>,
-    pending_values_map: BTreeMap<TxHash, Option<Instant>>,
+    pending_values_map: HashMap<TxHash, Option<Instant>>,
```

Comment:
i changed this because `HashMap` has a `retain` function and `BTreeMap` does not for some reason. AFAICT there is no reason to prefer either one here really

Reply:
It doesn't appear that the sorting is used anywhere - or at least the `pending_values_map` is not iterated over... one small nit -- can you change the comment above this line?